EasyVCR/TimeFrame.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+
+namespace EasyVCR
+{
+    /// <summary>
+    ///     TimeFrame used to store a number of days, hours, minutes and seconds.
+    /// </summary>
+    public class TimeFrame
+    {
+        /// <summary>
+        ///     Enums for common time frames.
+        /// </summary>
+        private enum CommonTimeFrames
+        {
+            Never,
```

Comment:
Question: If a cassette is never valid, why would they use VCR at all?

Question: Does it make sense to add something like 30, 60, 90, 180, 360 days here?

Reply:
1) I'm not sure, but I need to account for it.

2) I can build in more defaults, sure.

Reply:
Resolved